### ISSUE TITLE
Shiny app dataset explorer

### ISSUE THREAD

**Issue by kimberly-bastille** [NEFSC/NEFSC-Spatial#27]
Develop a shiny app that toggles datasets on and off. 

This could be a cool rollout tool for when we want to market this package to the masses.

Questions:
1) Where to store?
2) Who is developing? 
3) What type of features do we want the app to include?

**Comment by kimberly-bastille** [comment]
I like the idea of the app living in the package and [found this example](https://deanattali.com/2015/04/21/r-package-shiny-app/) which includes a function to run the app in the package (All in one delivery!). So the user would only have to run `NEFSCspatial::run_app()` to use the tool and explore the available datasets. I started a [shiny-dev branch](https://github.com/NEFSC/NEFSC-Spatial/tree/shiny-dev) where we can develop the app and function. 

Right now the app is just the "Old Faithful" default with a new name and is all in one script `app.R' but we can split that out if the code gets long.

**Comment by andybeet** [comment]
Exactly. I also have a [repo](https://github.com/andybeet/testapp) where i walked through the steps of turning just a repo containing the code for an app into a package to deploy the app
